pkg/ddevapp/drupal.go

```diff
@@ -93,7 +93,7 @@ $settings['file_scan_ignore_directories'] = [
 // This will prevent Drupal from setting read-only permissions on sites/default.
 $settings['skip_permissions_hardening'] = TRUE;
 
-// This is super ugly but it determines whether or not drush should include a custom settings file which allows
+// This determines whether or not drush should include a custom settings file which allows
```

Comment:
Oops, I think this appears 2 other places. Could you hit them too?

Reply:
I found just one more occurrence (d7), but it's gone now.